pkg/apis/apps/v1beta1/types.go

```diff
@@ -25,6 +25,10 @@ import (
 const (
 	// StatefulSetInitAnnotation if present, and set to false, indicates that a Pod's readiness should be ignored.
 	StatefulSetInitAnnotation = "pod.alpha.kubernetes.io/initialized"
+	// StatefulsetBurstAnnotation instructs the stateful set controller to not wait when replacing pods from the set,
```

Comment:
Doesn't respect ordered termination either. Basically it preserves uniqueness but not ordering.